<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KAIS open data convertor</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;thisdsds&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>781</width>
     <height>24</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>321</width>
     <height>181</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Търсене на имот/сграда</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Слой</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="combo_LayerSelect">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToMinimumContentsLength</enum>
      </property>
      <property name="minimumContentsLength">
       <number>0</number>
      </property>
      <property name="placeholderText">
       <string>Посочи слой за търсене на имот/сграда</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Идентификатор</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_CadastralID">
      <property name="placeholderText">
       <string>Въведете идентификатор на имот</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_FindParcel">
      <property name="toolTip">
       <string>Натиснете, за да откриете и селектирате имота по въведения идентификатор</string>
      </property>
      <property name="text">
       <string>Намери имот</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>321</width>
     <height>120</height>
    </rect>
   </property>
   <property name="title">
    <string>Експорт опции</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Избери формат</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="combo_ExportFormat">
      <property name="toolTip">
       <string>Изберете формат</string>
      </property>
      <item>
       <property name="text">
        <string>DXF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CSV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>KML</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_Export">
      <property name="toolTip">
       <string>Експортира избраният имот в избраният формат</string>
      </property>
      <property name="text">
       <string>Експортирай</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="plugin_info">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>421</width>
     <height>301</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>150000</width>
     <height>150000</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Извличане на геометрия от КАИС отворени данни </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
